### DISCUSSION TITLE
Vercel deployment with sanity: Cannot find module 'sanity' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by simon-bonnedahl** [vercel/next.js#44451] [Help] [answered]
Anyone got an idea why I have this issue?
<img width="1201" alt="SCR-20221230-qvx" src="https://user-images.githubusercontent.com/32783686/210101252-1f9ea30b-0701-4e95-807c-0eb832b1e05a.png">

**Comment by simon-bonnedahl** [comment, marked as answer]
Found a solution which was to make a .vercelignore file and add "sanity" in there to ignore the whole folder.

**Reply by ayushsgithub** [reply]
How to make .vercelignore file and add "sanity"

**Reply by simon-bonnedahl** [reply]
Write sanity in plain text only in a .vercelignore file in root
<img width="712" alt="SCR-20230417-mp1" src="https://user-images.githubusercontent.com/32783686/232514354-24f372af-35ac-4fea-927e-0d78e194caf9.png">

**Reply by ayushsgithub** [reply]
Thanks @simon-bonnedahl

**Comment by alexandermckay** [comment]
Add a prebuild script to your package.json file:

```
"prebuild": "npm i --prefix <cms directory name>"
```

Confirm it is working locally with `npm run build` and look out for the prebuild script executing
